packages/mobile/src/send/SendAmount.tsx

```diff
@@ -275,10 +275,10 @@ function SendAmount(props: Props) {
   }, [recipientVerificationStatus, addressValidationType, dollarAmount, getTransactionData, origin])
 
   const onRequest = React.useCallback(() => {
-    if (dollarAmount.isGreaterThan(DAILY_PAYMENT_LIMIT_CUSD)) {
+    if (dollarAmount.isGreaterThan(DEFAULT_DAILY_PAYMENT_LIMIT_CUSD)) {
```

Comment:
Does this constraint still make sense? With KYC in place, it will become increasingly unlikely that everyone will have the same payment limits. Perhaps the limit should be whatever the user's own limit is rather than assuming the default amount?

Reply:
The other person would have to actually send the funds. If the amount requested is higher than their limit it would probably fail, right? I may be missremembering how the request flow goes exactly.

Reply:
Oh yeah you're right. We skip the `SendAmount` screen for payment requests, which is where the check happens. What you have is probably the best solution for now.